Momento/Incubating/SimpleCacheClient.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using MomentoSdk.Incubating.Responses;
+
+namespace MomentoSdk.Incubating
+{
+    public class SimpleCacheClient : MomentoSdk.Responses.SimpleCacheClient
+    {
+        public SimpleCacheClient(string authToken, uint defaultTtlSeconds) : base(authToken, defaultTtlSeconds)
+        {
+        }
+
+        public SimpleCacheClient(string authToken, uint defaultTtlSeconds, uint dataClientOperationTimeoutSeconds) : base(authToken, defaultTtlSeconds, dataClientOperationTimeoutSeconds)
+        {
+        }
+
+        // TODO: Dictionary can be <string|byte[], string|byte[]>
+        public CacheDictionarySetResponse DictionarySet(string cacheName, string dictionaryName, Dictionary<string, string> dictionary)
```

Comment:
@malandis Could we also do `DictionarySet(string cacheName, string dictionaryName, Dictionary<string, byte[]> dictionary)` method. i think having string only for keys is ok to start but allowing byte values seems valuable. WDUT

Reply:
chatted on zoom more remembering previous convo now. Going to keep string string to start and get some customer feed back.